plugins/polio/api/vaccines/supply_chain.py

```diff
@@ -818,6 +818,14 @@ def to_html(self, request, queryset, view):
         return ""
 
 
+class RoundFilter(filters.BaseFilterBackend):
+    def filter_queryset(self, request, queryset, view):
+        round_id = request.query_params.get("round_id")
+        if round_id:
+            return queryset.filter(rounds=round_id)
+        return queryset
+
+
```

Comment:
When possible, we should use the `FilterSet` class, which will validate the query params under the hood.

example: https://github.com/BLSQ/iaso/blob/main/plugins/polio/api/chronogram/filters.py